docs/rfc/active/1778-dashboards/README.md

```diff
@@ -0,0 +1,117 @@
+- Start Date: 2020-08-03
+- RFC PR: https://github.com/linkedin/datahub/pull/1778
+- Implementation PR(s): https://github.com/linkedin/datahub/pull/1775
+
+# Dashboards
+
+## Summary
+
+Adding support for dashboards (and charts) metadata cataloging and enabling search & discovery for them.
+The design should accomodate for different dashboarding ([Looker](www.looker.com), [Redash](www.redash.io)) tools used within a company.
+
+## Motivation
+
+Dashboards are a key piece within a data ecosystem of a company. They are used by different groups of employees across different organizations.
+They provide a way to visualize some data assets (tracking datasets or metrics) by allowing slice and dicing of the input data source.
+When a company scales, data assets including dashboards gets richer and bigger. Therefore, it's important to find and access to the right dashboard.
+By having dashboards as a top level entity in Datahub, we achieve below goals:
+
+- Enabling Search & Discovery for dashboard assets
+- Link dashboards to underlying data sources and have a more complete picture of data lineage
+
+## Detailed design
+
+![high level design](high_level_design.png)
+
+As shown in the above diagram, dashboards are composed of a collection of charts at a very high level. These charts
+could be shared by different dashboards. In the example sketched above, `Chart_1`, `Chart_2` and `Chart_3` are part of
+`Dashboard_A` and `Chart_3` and `Chart_4` are part of `Dashboard_B`.
+
+### Entities
+There will be 2 top level GMA [entities](../../../what/entity.md) in the design: dashboards and charts.
+It's important to make charts as a top level entity because charts could be shared between different dashboards.
+We'll need to build `Contains` relationships between Dashboard and Chart entities.
```

Comment:
One drawback of leveraging `Contains` relationship is that it is generic enough for any `Has-A` relationship and contains no additional metadata. But what if we want to attach additional metadata to this edge between Dashboard and Chart entities?

Reply:
> One drawback of leveraging `Contains` relationship is that it is generic enough for any `Has-A` relationship and contains no additional metadata. But what if we want to attach additional metadata to this edge between Dashboard and Chart entities?

Extra attributes can be attached to a relationship, contains model can evolve in a backward compatible way.

Reply:
+1 to @camelliazhang. Edges can hold attributes as well.